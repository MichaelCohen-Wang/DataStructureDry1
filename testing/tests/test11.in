add_ship 126851 11
remove_ship 126851
add_pirate 484149 126851 14
get_cannons 126851
add_pirate 131050 401304 -1
update_pirate_treasure 484149 1
add_ship 521590 20
add_pirate 325199 521590 1
add_pirate 436621 521590 8
get_cannons 521590
update_pirate_treasure 484149 11
get_cannons 521590
remove_ship 126851
get_treasure 436621
update_pirate_treasure 436621 5
update_pirate_treasure 325199 27
add_ship 343859 18
update_pirate_treasure 436621 0
update_pirate_treasure 436621 10
get_treasure 325199
get_cannons 343859
add_pirate 196186 126851 15
update_pirate_treasure 436621 24
update_pirate_treasure 325199 27
add_ship 183985 0
add_ship 283175 8
remove_ship 521590
add_ship 175892 10
add_pirate 112384 343859 12
get_cannons 521590
get_treasure 436621
get_cannons 183985