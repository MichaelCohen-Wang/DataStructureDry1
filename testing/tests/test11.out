add_ship: SUCCESS
remove_ship: SUCCESS
add_pirate: FAILURE
get_cannons: FAILURE
add_pirate: FAILURE
update_pirate_treasure: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 20
update_pirate_treasure: FAILURE
get_cannons: SUCCESS, 20
remove_ship: FAILURE
get_treasure: SUCCESS, 8
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
get_treasure: SUCCESS, 28
get_cannons: SUCCESS, 18
add_pirate: FAILURE
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 20
get_treasure: SUCCESS, 47
get_cannons: SUCCESS, 0
